src/app/_components/DemoHero/DemoHero.js

```diff
@@ -0,0 +1,19 @@
+"use client";
```

Comment:
Should we make it client component just to use styles?

Reply:
Unfortunately, yes. It's the only way styled components could be used in nextjs. Removing "use client" brings up the following error:

Server Error: createContext only works in Client Components. Add the "use client" directive at the top of the file to use it. Read more: https://nextjs.org/docs/messages/context-in-server-component